setup.py

```diff
@@ -14,7 +14,7 @@ def read(f):
     return open(os.path.join(os.path.dirname(__file__), f)).read().strip()
 
 
-install_requires = ['aiohttp']
+install_requires = ['aiohttp', 'ipaddress']
```

Comment:
Python3 has this in standard library, I think we do not need to include `ipaddress`
https://docs.python.org/3/library/ipaddress.html